communicator/remote/command.go

```diff
@@ -25,7 +25,7 @@ type Cmd struct {
 	Stderr io.Writer
 
 	// Once Wait returns, his will contain the exit code of the process.
-	exitStatus int
+	ExitStatus int
```

Comment:
I think it would be better to keep this field private and instead add a `ExitStatus()` method that returns the `exitStatus`. Not sure if it really matters that much, but think it provides a slightly better (safer) API.

Reply:
After looking at the code again, I should be able to pull the exit code from the error, ExitError looks to return it.

Not sure why I put this here in the first place.